src/librustc_codegen_ssa/back/link.rs

```diff
@@ -583,6 +589,11 @@ fn link_natively<'a, B: ArchiveBuilder<'a>>(
         cmd.env_remove(k);
     }
 
+    // To parse error messages, l10n has to be turned off
+    cmd.env_remove("LANG");
+    cmd.env_remove("LC_MESSAGES");
+    cmd.env_remove("LC_ALL");
```

Comment:
It's a bit unfortunate that we lose localization of linker messages, but it's pretty much unavoidable. I think it's not such a big issue anyways since rustc only outputs English anyways.

Reply:
One could argue it makes searching for the solution online easier.